Fill out README.rst with instructions to build docs and pointer to online version. Refs issue #1

LeMeteore opened the pull request:
Let me know if I should change something. Also, advices are welcome concerning the built of pdf documentation. I still have some strange error after the build, and the result is a pdf file with basically 6 blank pages :(

Commit:
Fill out README.rst with instructions to build docs and pointer to online version. Refs issue #1

webmaven commented on README.rst:
```diff
@@ -10,8 +10,88 @@ A short tutorial, suitable for beginners. See the `online version`_.
+--------------------------------------------
+`Virtualenv`_ will help you creates Python installations that doesn’t share libraries either with the globally installed packages, either with the other virtual environments. Let's create a virtualenv for morepath, just type inside your terminal::
+
+      $ virtualenv morepath_env
```
This should probably be a separate venv than the one we're going to use for the tutorial. Perhaps `morepath_docs_env`, or do you think that's too long?

LeMeteore replied:
Too long? No. A separate env named morepath_docs_env, ok.

webmaven commented on README.rst:
```diff
@@ -10,8 +10,88 @@ A short tutorial, suitable for beginners. See the `online version`_.
+.. _pip:
+   http://pip.readthedocs.org/en/latest/quickstart.html
+
+If everything went well, you should read this message ``Successfully installed Jinja2 MarkupSafe Pygments Sphinx WebOb docutils future morepath reg repoze.lru sphinx-bootstrap-theme venusifork Cleaning up...``. Otherwise, you may have done something wrong.
```
I will probably reduce the required packages to only the docs related ones at some point. If you would like to clean that up in a later PR, that would also be welcome.

LeMeteore replied:
Reduce the required packages to only the docs related one. So you are talking about all the packages installed after a "pip install sphinx", basically: Jinja2==2.7.3
MarkupSafe==0.23, Pygments==2.0.1, Sphinx==1.2.3, argparse==1.2.1, docutils==0.12, wsgiref==0.1.2 Ok, no problem.

webmaven commented on README.rst:
```diff
@@ -10,8 +10,88 @@ A short tutorial, suitable for beginners. See the `online version`_.
+------------------------
+To build the html documentation, just go to the docs folder and type::
+
+      $ cd docs; make html
```
split into two lines.

LeMeteore replied:
one line for "cd docs". Another line for "make html". Ok :)

webmaven commented on README.rst:
```diff
@@ -10,8 +10,88 @@ A short tutorial, suitable for beginners. See the `online version`_.
+
+Build html documentation
+------------------------
+To build the html documentation, just go to the docs folder and type::
```
'go into the docs folder' repeats the content of the command, this is OK, as long as you interleave the commands with the prose.

LeMeteore replied:
You are too kind, but there is no problem, I'll change it.

webmaven commented on README.rst:
```diff
@@ -10,8 +10,88 @@ A short tutorial, suitable for beginners. See the `online version`_.
+All the html pages will be generated and available under the _build/html/ folder. Inside this folder, there is an index.html file that can be called to open the morepath documentation home page.
+
+
+Build pdf documentation
```
As you noted, this doesn't currently work. I'll open a separate issue for the problem, and see what I can figure out. Meanwhile lets keep the section.

LeMeteore replied:
Ok.

webmaven commented on README.rst:
```diff
@@ -10,8 +10,88 @@ A short tutorial, suitable for beginners. See the `online version`_.
    http://morepath-tutorial.readthedocs.org/en/latest/
-   
+
 .. |docs| image:: https://readthedocs.org/projects/morepath-tutorial/badge/?version=latest
```
Move these definitions for `online version` and `|docs|` to the end of the document.

LeMeteore replied:
Ok.

webmaven commented on README.rst:
```diff
@@ -10,8 +10,88 @@ A short tutorial, suitable for beginners. See the `online version`_.
+      $ sudo aptitude install build-essential python-dev texlive-full
+
+
+This is the first option, probably not the best way to do, as there is not an implicit need of all the packages that will be installed. After this install, you'll have the necessary packages to build a pdf documentation. Normally, you are still inside the docs folder. Just type in your terminal::
```
Can we cut down the list to just the ones that are needed, and remove the 'not an implicit need' caveat?

LeMeteore replied:
This is actually the ones needed. But, texlive-full will bring a huge list of dependencies (that will not be an implicit need). Install texlive alone, and it won't work. I'm actually looking for an alternative. I'll let you know. New PR in the next few hours.

Commit:
Update of README.rst with instructions to build docs & pointer to online version. Refs issue#1

Commit:
Documentation: list of packages needed.

The docs_requirements.txt file will be used to list only packages we need to
build the morepath documentation.

Refs issue#1

webmaven commented on README.rst:
```diff
@@ -8,10 +8,100 @@ Purpose
 
+
+Instructions to build docs
+--------------------------
```
All the headers _after_ this one should be demoted one level.

webmaven commented on README.rst:
```diff
@@ -8,10 +8,100 @@ Purpose
+Instructions to build docs
+--------------------------
+
+You'll find the documentation homepage at: http://morepath-tutorial.readthedocs.org/en/latest/
```
No need for this line.

webmaven commented on README.rst:
```diff
@@ -8,10 +8,100 @@ Purpose
+--------------------------
+
+You'll find the documentation homepage at: http://morepath-tutorial.readthedocs.org/en/latest/
+But if like me you want to build your own doc, for an offline use, just follow the next steps:
```
"To build your own docs for offline use, just..." etc.

webmaven commented on README.rst:
```diff
@@ -8,10 +8,100 @@ Purpose
+     $ make html
+
+
+All the html pages will be generated and available under the _build/html/ folder. Inside this folder, there is an index.html file that can be called to open the morepath documentation home page.
```
`_build/html/`

Commit:
Fill out README.rst with instructions to build documentation & pointer to online version. Refs issue #1

LeMeteore commented:
I sent another PR, I close this one.
